From e2012b50883b3c7f3aa1c4aab07613757368e846 Mon Sep 17 00:00:00 2001
From: Aleksandr Korzun <a.korzun@partner.samsung.com>
Date: Mon, 10 Apr 2023 17:23:20 +0300
Subject: [PATCH 101/225] [axdimm_resource] Add build instructions to
 PNM_HOW_TO_BUILD.md

Add instructions for building and installing the `axdimm_resource` module
to `PNM_HOW_TO_BUILD.md`.

References: MCS23-693

Signed-off-by: Aleksandr Korzun <a.korzun@partner.samsung.com>
---
 PNM_HOW_TO_BUILD.md | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/PNM_HOW_TO_BUILD.md b/PNM_HOW_TO_BUILD.md
index dd3d52729..964805de4 100644
--- a/PNM_HOW_TO_BUILD.md
+++ b/PNM_HOW_TO_BUILD.md
@@ -37,6 +37,17 @@ Built drivers can then be installed with `install_dax_drivers` script:
 $ ./install_dax_drivers.sh
 ```
 
+# AXDIMM resource driver only
+
+```bash
+$ make M=drivers/pnm/axdimm_resource/ -j32
+```
+
+Built drivers can then be installed with `install_axdimm_resource` script:
+```bash
+$ ./install_axdimm_resource.sh
+```
+
 # DBA resource driver only
 
 ```bash
@@ -47,3 +58,8 @@ Built drivers can then be installed with `install_dba_resource` script:
 ```bash
 $ ./install_dba_resource.sh
 ```
+
+# Install AXDIMM headers only
+```bash
+$ ./install_axdimm_headers.sh
+```
-- 
2.34.1

